contracts/staking/l2_contracts/SymbioticStaking.sol

```diff
@@ -199,15 +182,15 @@ contract SymbioticStaking is
 
         _verifyProof(_imageId, STAKE_SNAPSHOT_TYPE, _index, _numOfTxs, _captureTimestamp, _vaultSnapshotData, _proof);
 
-        // update Vault and Operator stake amount
-        // update rewardPerToken for each vault and operator in SymbioticStakingReward
+        // update Vault and Prover stake amount
+        // update rewardPerToken for each vault and prover in SymbioticStakingReward
         _submitVaultSnapshot(_captureTimestamp, _vaultSnapshots);
 
         _updateTxCountInfo(_numOfTxs, _captureTimestamp, STAKE_SNAPSHOT_TYPE);
 
         // when all chunks of VaultSnapshots are submitted
         if (_index == _numOfTxs - 1) {
-            submissionStatus[_captureTimestamp][msg.sender] |= STAKE_SNAPSHOT_MASK;
+            submissionStatus[_captureTimestamp][msg.sender] = Enum.SubmissionStatus.STAKE_SNAPSHOT_DONE;
```

Comment:
Stake snapshot and slash snapshot being sequential isn't ideal. We probably have to allow them in any order.

Reply:
What if they aren't submitted in order?

Reply:
Resolved